### PR TITLE
Add the deallocated server state

### PR DESCRIPTION
**Pull request by jackwaudby**
Add the `DEALLOCATED` server state to "Managing servers in a cluster".

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates.

You can view the updated docs at https://neo4j-docs-operations-967.surge.sh

File: modules/ROOT/pages/clustering/servers.adoc [hunk 1]
```diff
@@ -84,6 +85,11 @@ When a server is no longer needed, it cannot be removed from the cluster while i
 The command `DEALLOCATE DATABASE[S] FROM SERVER[S] _server_[,...]` is used to transition servers to the _Deallocating_ state, reallocating all their hosted databases to other servers in the cluster.
 Additionally, servers which are deallocating will not have any further databases allocated to them.
 
+=== Deallocated state
+
+When a server is in the deallocated state it no longer hosts any databases any databases besides system and can be removed from the cluster.
+Additionally, deallocated servers will not have any further databases allocated to them.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/clustering/servers.adoc:91]:
```suggestion
Additionally, deallocated servers cannot have any further databases allocated to them.
```

**Review by AlexicaWright — COMMENTED** [review]
Just one small tweak. Looks good @jackwaudby , thanks!

**Comment by jackwaudby** [comment]
Awesome 😎 please merge away!

**Review by AlexicaWright — APPROVED** [review]
👍

**Pull request merged**